tests/rules/javascript/test.content_scripts_file_absent.js

```diff
@@ -27,12 +25,26 @@ describe('content_scripts_file_absent', () => {
       const jsScanner = createJsScanner(code, fileRequiresContentScript);
 
       const { linterMessages } = await jsScanner.scan();
+
       expect(linterMessages.length).toEqual(1);
       expect(linterMessages[0].code).toEqual(CONTENT_SCRIPT_NOT_FOUND.code);
       expect(linterMessages[0].type).toEqual(VALIDATION_ERROR);
     });
   });
 
+  it('should not show an error when relative content script is missing', async () => {
```

Comment:
Nit, we could probably merge this test with the one at line 68 (especially given that the new version of this rule never validates relative content script paths, without any difference between being in a top level js file or a js file from a sub directory).